env_cylc.py

```diff
@@ -115,33 +110,29 @@ def get_tseries_resubmit(self, ts_tper, ts_n, stop_n, stop_option):
             freq = (ts/cesm)+1
         else:
             freq = (ts/cesm)
-        return freq  
-               
-         
+        return freq
+
+
 
     def get_env(self):
         my_case = os.getcwd() + '/../'
-        
+
         case = Case(my_case, read_only=False)
-        
+
         cwd = os.getcwd()
         os.chdir(my_case)
 
-        env_case = EnvCase()
-        machine_name = env_case.get_value('MACH')
+        machine_name = case.get_value('MACH')
         print 'Running on ',machine_name
 
-        env_mach = Machines(machine=machine_name)
-        batch_system = env_mach.get_value("BATCH_SYSTEM")
+        batch_system = case.get_value("BATCH_SYSTEM")
         batch = Batch(batch_system=batch_system, machine=machine_name)
-        env_run = EnvRun()
-        env_build = EnvBuild() 
         env_batch = case.get_env("batch")
         os.system('./xmlchange RESUBMIT=0')
```

Comment:
You have a case object, instead of this os.system call you can do 
case.set_value(RESUBMIT, 0)